src/lem/tests/eval_tests.rs

```diff
@@ -2792,9 +2794,13 @@ fn test_eval_non_symbol_binding_error() {
 
 #[test]
 fn test_dumb_lang() {
-    use crate::{
-        coprocessor::test::DumbCoprocessor, eval::tests::coproc::DumbCoproc, state::user_sym,
-    };
+    use crate::{self as lurk};
+    use crate::{coprocessor::test::DumbCoprocessor, state::user_sym};
+
+    #[derive(Clone, Debug, Coproc)]
+    enum DumbCoproc<F: LurkField> {
+        DC(DumbCoprocessor<F>),
+    }
```

Comment:
Can we simply not define this and use `DumbCoprocessor` directly? It's being redefined in `src/proof/tests/nova_tests_lem.rs`